designer/client/src/Component.tsx

```diff
@@ -190,6 +190,30 @@ export const Html: FunctionComponent = () => {
   )
 }
 
+export const FileUploadField: FunctionComponent = () => {
+  return (
+    <ComponentField>
+      <div className="box">
+        <div className="fileUploadIconContainer" aria-hidden="true">
```

Comment:
Can we talk about `aria-hidden="true"`?

I've added some suggestions to make sure the SVG is announced but we might be able to solve how all components are announced (including all the non-SVG ones)

Reply:
Yeah got 2x correct answers here

Can either announce the icon (like for search) or hide it and add text another way

E.g. Update **SearchIcon.jsx** with `aria-hidden="true" focusable="false"` since we now have:

* https://github.com/DEFRA/forms-designer/pull/326

Reply:
Happy with the title suggestion as per SearchIcon.jsx, I've stuck with that. I didn't realise it was there before.